static/app/components/sidebar/broadcastSdkUpdates.tsx

```diff
@@ -26,18 +32,83 @@ const flattenSuggestions = (list: ProjectSdkUpdates[]) =>
     []
   );
 
-const BroadcastSdkUpdates = ({projects, sdkUpdates}: Props) => {
+function BroadcastSdkUpdates({projects, sdkUpdates, organization}: Props) {
   if (!sdkUpdates) {
     return null;
   }
 
   // Are there any updates?
-  if (flattenSuggestions(sdkUpdates).length === 0) {
+  if (!flattenSuggestions(sdkUpdates).length) {
     return null;
   }
 
-  // Group SDK updates by project
-  const items = Object.entries(groupBy(sdkUpdates, 'projectId'));
+  function renderUpdates(projectSdkUpdates: ProjectSdkUpdates[]) {
+    // Group SDK updates by project
+    const items = Object.entries(groupBy(projectSdkUpdates, 'projectId'));
+
+    return items
+      .map(([projectId, updates]) => {
+        const project = projects.find(p => p.id === projectId);
+
+        if (!project) {
+          return null;
+        }
+
+        // Updates should only be shown to users who are project members or users who have open membership or org write permission
+        const hasPermissionToSeeUpdates =
+          project.isMember ||
+          organization.features.includes('open-membership') ||
+          organization.features.includes('org:write');
```

Comment:
features will never include `org:write`. I think you want `organization.access` here instead.

Reply:
oh good catch!! thanks